Makefile

```
@@ -35,7 +35,8 @@ GO_BUILD_FLAGS ?= -ldflags '$(LDFLAGS)'
 MM_UTILITIES_DIR ?= ../mattermost-utilities
 DLV_DEBUG_PORT := 2346
 MATTERMOST_PLUGINS_PATH=$(MM_SERVER_PATH)/plugins
-FOCALBOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/focalboard
+BOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/board
+PLUGIN_NAME=board
```

Should be `Boards` plural

```suggestion
PLUGIN_NAME=boards
```

---

Done.